custom_components/luxtronik/sensor_entities_predefined.py

```diff
@@ -330,10 +330,10 @@
         key=SensorKey.CURRENT_HEAT_OUTPUT,
         luxtronik_key=LC.C0257_CURRENT_HEAT_OUTPUT,
         icon="mdi:lightning-bolt-circle",
-        state_class=SensorStateClass.MEASUREMENT,
+        state_class=SensorStateClass.TOTAL_INCREASING,
         device_class=SensorDeviceClass.ENERGY,
         entity_category=EntityCategory.DIAGNOSTIC,
-        native_unit_of_measurement=UnitOfPower.WATT,
+        native_unit_of_measurement=UnitOfEnergy.KILO_WATT_HOUR,
```

Comment:
Can we simple change from Watt to KiloWatt?
My heatpump doesn't have this sensor. But I think if luxtronik sends the value in Watt we have to set a factor: 0.001

Reply:
Yes

Watt is not a valid unit of measurement for Energy. It should be Wh or kWh.
![image](https://github.com/BenPru/luxtronik/assets/5879533/ca548a00-fbd7-430c-8a33-ada14a4278c3)
In this case, it is KwH, just like the other sensors with SensorDeviceClass.ENERGY

The sensors are currently 0 due to another bug (probably related to the external temperature).